src/app/core/store/collections/collection.util.ts

```diff
@@ -22,7 +22,11 @@ import {Attribute, Collection} from './collection';
 export function isAttributeEditable(attributeId: string, collection: Collection): boolean {
   const attribute =
     attributeId && collection && collection.attributes && collection.attributes.find(attr => attr.id === attributeId);
-  return attribute && (!attribute.function || !attribute.function.js || attribute.function.editable);
+  return isAttributeEditable_(attribute);
+}
+
+export function isAttributeEditable_(attribute: Attribute): boolean {
```

Comment:
I don't like this suffix :( 
I would suggest you to rename first function to `isCollectionAttributeEditable` and problem will be solved :)